src/packages/cli/src/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -4,18 +4,16 @@ exports[`should work with a custom output dir 1`] = `
 
 Prisma schema loaded from prisma/schema.prisma
 
-✔ Generated Prisma Client (version: 0.0.0) to ./generated/client in XXms
-
-You can now start using Prisma Client in your code:
-
+✔ Generated Prisma Client (2.11.0-dev.20) to ./generated/client in XXms
```

Comment:
Not a problem that a concrete version is mentioned here now?

Reply:
If it's true it's gonna be fun to update the snapshots 😅